default-tiles/status-bar/src/first_line.rs

```diff
@@ -70,90 +70,64 @@ fn unselected_mode_shortcut(letter: char, text: &str) -> LinePart {
         .fg(BLACK)
         .on(BRIGHT_GRAY)
         .bold()
-        .paint(format!(" <"));
+        .paint(" <");
     let char_shortcut = Style::new()
         .bold()
         .fg(RED)
         .on(BRIGHT_GRAY)
         .bold()
-        .paint(format!("{}", letter));
+        .paint(letter.to_string());
     let char_right_separator = Style::new()
         .bold()
         .fg(BLACK)
         .on(BRIGHT_GRAY)
         .bold()
-        .paint(format!(">"));
-    let styled_text = Style::new()
-        .fg(BLACK)
-        .on(BRIGHT_GRAY)
-        .bold()
-        .paint(format!("{} ", text));
+        .paint(">");
+    let styled_text = Style::new().fg(BLACK).on(BRIGHT_GRAY).bold().paint(text);
```

Comment:
Didn't we drop a padding character here?

Reply:
Good catch on these! Sorry! I was getting a bit overzealous here :sweat_smile:

Reply:
Cool :) I forgot to mention that I didn't run this, I just commented on the code itself. So if you could run it to make sure there's no visual change (just to be sure I caught everything) that would be great :)

Reply:
Strangely, I tested it quite a bit without noticing! I've done a side-by-side with `main` now though and it's looking that same!